### DISCUSSION TITLE
Rotate bar chart 90 deg?

### DISCUSSION THREAD

**Discussion by abathur8bit** [imaNNeo/fl_chart#1065] [Q&A]
My client is looking for a bar chart, like the example in [bar_chart_sample2.dart](https://github.com/imaNNeoFighT/fl_chart/blob/master/example/lib/bar_chart/samples/bar_chart_sample2.dart). But wants to rotate it 90 deg and have a large list of items. For example, the attached image would contain 80+ items, and would be scrollable (up and down). 

![image](https://user-images.githubusercontent.com/24625148/174315893-7f64ba90-97bc-4161-a78b-6cb15833d3d5.png)

Is there a way to rotate the chart? Can it handle a large number of items? Would I be better off creating a bunch of smaller charts (assuming I could make them at 90 deg) and put them in a ListView?

**Comment by vishnuagbly** [comment]
Look at this #113

**Reply by imaNNeo** [reply]
Yes, you're right. We have an issue for this feature request.

**Reply by imaNNeo** [reply]
So I will close this discussion
